changelogs/fragments/397-he_engine_abrt_configuration.yaml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - HE deploy - configured abrt initial files only when needed (https://github.com/oVirt/ovirt-ansible-collection/pull/397).
+
```

Comment:
Just to keep the convention, add `---` in the first line and remove the last empty line.
Use the role name `hosted_engine_setup` instead of `HE deploy`.